psychopy/tools/pkgtools.py

```diff
@@ -240,27 +240,24 @@ def installPackage(
             and error from the subprocess.
         If `awaited=False`:
             Returns the job (thread) which is running the install.
-    """
-    if target is None:
-        target = prefs.paths['userPackages']
```

Comment:
I think it makes sense to first check `prefs.paths['userPackages']` trying to set `target`.